src/utils/urlParseHelper.ts

```diff
@@ -25,9 +35,9 @@ export interface IParsedUrl {
 }
 
 /**
- * Parses SCM Repository URI or SCP to webview link of the SCM Repository.
+ * Parses SCM Repository URI or SCP-like URI to webview link of the SCM Repository.
  *
- * For SCP, the parser will ignore any port number and regard the text between ':' and '/' as
+ * For SCP-like URI, the parser will ignore any port number and regard the text between ':' and '/' as
```

Comment:
SCP-like URIs do not and cannot contain any port. It is normal URLS that can contain them.

Reply:
You probably meant:
```
In SCP-like URIs, the text after ':' is never regarded as the port but as part of the path...
```

Reply:
updated